extensions.html

```diff
@@ -606,18 +606,20 @@ <h3>
               {{GamepadEffectParameters/weakMagnitude}}, none of which are
               required.
             </p>
-            <p>
-              {{GamepadEffectParameters/startDelay}} and
-              {{GamepadEffectParameters/duration}} set the duration of the
-              initial delay before after initiating the effect before the
-              effect will be started
-            </p>
             <p>
               {{GamepadEffectParameters/startDelay}} sets the duration of the
               delay after {{GamepadHapticActuator/playEffect()}} is called
-              until vibration is started, default 0. During the delay interval,
-              the actuator must not vibrate. An effect with negative
+              until vibration is started, in milliseconds. During the delay
+              interval, the actuator must not vibrate. An effect with negative
               {{GamepadEffectParameters/startDelay}} is not a [=valid effect=].
+              If the {{GamepadEffectParameters/startDelay}} is not specified,
+              there is no initial delay before vibration starts.
```

Comment:
I wonder if it's clearer to treat this as 0? Or am I misreading?

Reply:
It's probably best to describe it as a default value. Defining a default value makes it clear that undefined startDelay should behave the same as startDelay == 0.